cmd/cluster-agent/webhook/listener.go

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+/*
+Package api implements the agent IPC api. Using HTTP
+calls, it's possible to communicate with the agent,
+sending commands and receiving infos.
+*/
```

Comment:
```suggestion
// Package api implements the agent IPC api. Using HTTP
// calls, it's possible to communicate with the agent,
// sending commands and receiving infos.
```
This is more common.

Reply:
Addressed 👍